src/test/java/bio/terra/service/filedata/google/firestore/FileOperationTest.java

```diff
@@ -564,6 +577,11 @@ private BulkLoadRequestModel makeBulkFileLoad(String tagBase,
             for (int i = 0; i < fileCount; i++) {
                 BulkLoadFileModel fileModel = getFileModel(validPattern[i], startIndex + i, testId);
                 String fileLine = objectMapper.writeValueAsString(fileModel) + "\n";
+                // Inject extra key-value pairs into file lines
```

Comment:
The whole JSON string has to be on a single line. I think you need to move the ` + "\n"` to after the point where you inject the extra keys.

Reply:
Good catch!

Reply:
The whole JSON string should continue to be on a single line. The replace shouldn't touch the `+ "\n"` at the end of the string since it's pushing the extra key to the front of the string.